model_compression_toolkit/core/keras/hessian/trace_hessian_calculator_keras.py

```diff
@@ -48,4 +56,25 @@ def __init__(self,
                                                           trace_hessian_request=trace_hessian_request,
                                                           num_iterations_for_approximation=num_iterations_for_approximation)
 
+    def _concat_outputs(self, outputs: Union[tf.Tensor, List[tf.Tensor]]) -> tf.Tensor:
```

Comment:
same here. Create a general concat function (one use is for concat multiple model's output).

Reply:
Where would you think it should be? Since other parts outside the hessian package may use it, and we don't have a utils module (if I'm not wrong)